lib/core/bootstrap.class.ts

```diff
@@ -14,21 +14,47 @@ export function bootstrap(...BootstrapedApps: Array<IAppStatic>): Array<IApp> {
     const app: IApp = new BootstrapedApp(server)
 
     server.connection(_.extend(Reflect.getMetadata('hapiour:config', BootstrapedApp), DEFAULT_APP_CONFIG))
-    try {app.onInit()} catch(err) {}
+    try {
+      app.onInit()
+    } catch (err) {
+      _.noop()
+    }
     const plugins: Array<IPlugin> = getPluginsRecurs(Reflect.getMetadata('hapiour:plugins', BootstrapedApp))
     const routes: Array<RouteConfiguration> = getRoutesWithConfigRecurs(BootstrapedApp)
     if (_.isEmpty(plugins)) {
-      try {app.onRegister()} catch(err) {}
+      try {
```

Comment:
If the goal of the `try/catch` clause is only for checking that the methods are defined, maybe you can use `_.attempt` which will call the method only if defined.
Moreover, it will be more relevant as the `try/catch` may hide errors thrown by the method calls themselves and can be confusing.